src/test/java/dev/openfeature/sdk/EvalContextTest.java

```diff
@@ -184,7 +183,7 @@ public class EvalContextTest {
 
         ctx2.setTargetingKey("  ");
         ctxMerged = ctx1.merge(ctx2);
-        assertEquals(key1, ctxMerged.getTargetingKey());
```

Comment:
This is the only major change - we do not accept empty value or spaces for targeting key. Eaerlier this was handled in the merge starge where we filtered for null or empty values.